README.md

```diff
@@ -437,7 +438,6 @@ All these projects contain steps and resources required for reproduction.
 - [Saturday Science & BBC micro:bits](https://saturdayscience.org/bbc-microbit/) - Practical science and engineering projects with the micro:bit, explore physical properties with cool experiments.
 - [Maker Pro micro:bit Projects & Tutorials](https://maker.pro/microbit) - The micro:bit section of Maker Pro, a place for makers to share designs, collaborate, and learn how to take your product to market.
 
```

Comment:
```suggestion


```

There was two new lines before the Articles section, now there is only one blank line in this PR. So we need to add an extra line.

Reply:
Done.